xbmc/input/KeyboardLayout.h

```diff
@@ -56,4 +58,5 @@ class CKeyboardLayout
   std::string m_language;
   std::string m_layout;
   Keyboards m_keyboards;
+  IInputCodingTable *m_codingtable;
```

Comment:
so change this line to
```
std::unique_ptr<IInputCodingTable> m_codingtable;
```
and line 34 to:
```
  IInputCodingTable* GetCodingTable() { return m_codingtable.get(); }
```
this change cause compile error in `CGUIDialogKeyboardGeneric::UpdateButtons`:
```
error C2280: 'std::unique_ptr<IInputCodingTable,std::default_delete<_Ty>>::unique_ptr(const std::unique_ptr<_Ty,std::default_delete<_Ty>> &)' : attempting to reference a deleted function
```